docs/API/REST-V2-Usage.md

```diff
@@ -51,163 +55,4 @@ null value.
 
 Create custom notifications for email or Slack issues. 
 
-We are investigating moving this out of Evergreen (EVG-21065) and won't be supporting future work for this. 
-
-## Additional Endpoints
-
-### Users
-
-#### Endpoints
-
-##### Get Users for Role
-
-NOTE: These roles are not part of the OpenAPI spec, since they are part of another package.
```

Comment:
I feel like these were the routes you'd meant to keep up here https://github.com/evergreen-ci/evergreen/blob/739214c74f0eb04f1efe34d9fa9911aa6911f251/rest/route/service.go#L225, or were these never documented?

Reply:
They weren't documented. Only GET /roles/<role_id>/users was documented https://github.com/evergreen-ci/evergreen/blob/ac7582937dbc56e46a925857c72e25a2cca77a69/docs/REST-V2-Usage.md#give-roles-to-user.